docs/ui/components/Search/styles.ts

```diff
@@ -0,0 +1,862 @@
+      fill: ${theme.palette.primary[600]};
+      stroke: ${theme.palette.primary[600]};
+    }
+
+    .DocSearch-Hit[aria-selected='true'] .DocSearch-Hit-icon:before {
+      filter: brightness(1.25);
+    }
+  }
+
+  /* === Button === */
+
+  .DocSearch-Button {
+    align-items: center;
+    background: var(--docsearch-searchbox-background);
+    border: 0;
+    border-radius: ${borderRadius.small}px;
+    color: var(--docsearch-muted-color);
+    cursor: pointer;
+    display: flex;
+    font-weight: 500;
+    height: 40px;
+    justify-content: space-between;
+    margin: 0 ${spacing[3]}px 0;
+    padding: ${spacing[0.5]}px ${spacing[3]}px;
+    user-select: none;
+    width: 24vw;
+    max-width: 320px;
+    box-shadow: ${shadows.input};
+    border: 1px solid ${theme.border.default};
+  }
+
+  .DocSearch-Button:hover,
+  .DocSearch-Button:active,
+  .DocSearch-Button:focus {
+    color: var(--docsearch-text-color);
+    outline: none;
+  }
+
+  .DocSearch-Button-Container {
+    align-items: center;
+    display: flex;
+  }
+
+  .DocSearch-Search-Icon {
+    stroke-width: 1.6;
+  }
+
+  .DocSearch-Button .DocSearch-Search-Icon {
+    color: ${theme.icon.default};
+    width: ${iconSize.small}px;
+    margin-right: ${spacing[1]}px;
+  }
+
+  .DocSearch-Button-Placeholder {
+    font-size: 1rem;
+    padding: 0 12px 0 6px;
+    color: ${theme.text.secondary};
+    opacity: 0.66;
+  }
+
+  .DocSearch-Button-Keys {
+    display: flex;
+    min-width: calc(2 * 20px + 2 * 0.4em);
+  }
+
+  .DocSearch-Button-Key {
+    display: flex;
+    align-items: center;
+    height: 20px;
+    ${typography.utility.pre}
+    ${kbdStyle}
+    margin-left: 0.4em;
+  }
+
+  @media (max-width: 768px) {
+    .DocSearch-Button {
+      width: 42px;
+    }
+
+    .DocSearch-Button-Keys,
+    .DocSearch-Button-Placeholder {
+      display: none;
+    }
+  }
+
+  /* === Modal === */
+  /* Body modifier */
+
+  .DocSearch--active {
+    overflow: hidden !important;
+  }
+
+  .DocSearch--active #__next {
+    filter: blur(6px);
+  }
+
+  /* Container & Modal */
+
+  .DocSearch-Container,
+  .DocSearch-Container * {
+    box-sizing: border-box;
+  }
+
+  .DocSearch-Container {
+    background-color: rgba(0, 0, 0, 0.33);
+    height: 100vh;
+    left: 0;
+    position: fixed;
+    top: 0;
+    width: 100vw;
+    z-index: 200;
+  }
+
+  .DocSearch-Container a {
+    text-decoration: none;
+  }
+
+  .DocSearch-Link {
+    appearance: none;
+    background: none;
+    border: 0;
+    color: var(--docsearch-highlight-color);
+    cursor: pointer;
+    font: inherit;
+    margin: 0;
+    padding: 0;
+  }
+
+  .DocSearch-Modal {
+    background: var(--docsearch-modal-background);
+    border-radius: 6px;
+    box-shadow: var(--docsearch-modal-shadow);
+    flex-direction: column;
+    margin: 72px auto auto;
+    max-width: var(--docsearch-modal-width);
+    min-width: 680px;
+    position: relative;
+    border: 1px solid ${theme.border.default};
```

Comment:
```suggestion
    border: 1px solid ${theme.border.default};
    border-radius: ${borderRadius.large}px;
```
This modal deserves a larger border radius